rbc/src/main/java/com/library/rbc/model/Suggestion.java

```diff
@@ -0,0 +1,28 @@
+package com.library.rbc.model;
+
+import com.library.rbc.model.enums.SuggestionStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.LocalDateTime;
+
+@Document("suggestions")
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Suggestion {
+
+    @Id
+    private String id;
+    private SuggestedBook suggestedBook;
+    private SuggestionStatus suggestionStatus;
+    private String orderFrom;
+    private Integer amount;
+    private LocalDateTime creationDate;
+    private String description;
+}
```

Comment:
The `SuggestedBook` class represents duplicated information about books that are stored with every `Suggestion`.  The information about who created the `Suggestion` (i.e. user id) is not information about the book itself, so it makes more sense to store the id of the user who created the `Suggestion` as a top-level field in the `Suggestion` class (e.g. `createdBy`).